aws/resource_aws_glue_crawler_test.go

```diff
@@ -1391,6 +1516,146 @@ resource "aws_glue_crawler" "test" {
+}
+
+func testAccGlueCrawlerConfig_CatalogTarget_TableList(tables []string) string {
+	var list string
+	for _, table := range tables {
+		list += fmt.Sprintf(`"${aws_glue_catalog_table.%s.name}",`, table)
+	}
+
+	return list
+}
+
+func testAccGlueCrawlerConfig_CatalogTarget(rName string, tables []string) string {
+	tableResources := testAccGlueCrawlerConfig_CatalogTarget_TableResources(tables)
+	catalogTableList := testAccGlueCrawlerConfig_CatalogTarget_TableList(tables)
+
+	return testAccGlueCrawlerConfig_Base(rName) + fmt.Sprintf(`
+resource "aws_glue_catalog_database" "test" {
+  name = %q
+}
+
+resource "aws_s3_bucket" "default" {
+  bucket = %q
+  force_destroy = true
+}
+
+%s
+
+resource "aws_glue_crawler" "test" {
+  depends_on = ["aws_iam_role_policy_attachment.test-AWSGlueServiceRole"]
+
+  database_name = "${aws_glue_catalog_database.test.name}"
+  name          = %q
+  role          = "${aws_iam_role.test.name}"
+
+  schema_change_policy {
+    delete_behavior = "LOG"
+  }
+  
+  catalog_target {
+    database_name = "${aws_glue_catalog_database.test.name}"
+    tables = [%s]
+  }
+
+  configuration = <<EOF
+{
+  "Version":1.0,
+  "Grouping": {
+    "TableGroupingPolicy": "CombineCompatibleSchemas"
+  }
+}
+EOF
+}
+`, rName, rName, tableResources, rName, catalogTableList)
+}
+
+func testAccGlueCrawlerConfig_CatalogTarget_Multiple(rName string) string {
+	return testAccGlueCrawlerConfig_Base(rName) + fmt.Sprintf(`
+resource "aws_glue_catalog_database" "test_1" {
+  name = "%s_test_1"
+}
+
+resource "aws_glue_catalog_database" "test_2" {
+  name = "%s_test_2"
+}
+
+resource "aws_glue_catalog_table" "test_1" {
+  database_name = "${aws_glue_catalog_database.test_1.name}"
+  name          = "table_1"
+  table_type    = "EXTERNAL_TABLE"
+
+  storage_descriptor {
+    location      = "s3://${aws_s3_bucket.default.bucket}"
+  }
+}
+
+resource "aws_glue_catalog_table" "test_2" {
+  database_name = "${aws_glue_catalog_database.test_2.name}"
+  name          = "table_2"
+  table_type    = "EXTERNAL_TABLE"
+
+  storage_descriptor {
+    location      = "s3://${aws_s3_bucket.default.bucket}"
+  }
+}
+
+resource "aws_s3_bucket" "default" {
+  bucket = %q
+  force_destroy = true
+}
+
+resource "aws_glue_crawler" "test" {
+  depends_on = ["aws_iam_role_policy_attachment.test-AWSGlueServiceRole"]
+
+  database_name = "${aws_glue_catalog_database.test_1.name}"
+  name          = %q
+  role          = "${aws_iam_role.test.name}"
+
+  schema_change_policy {
+    delete_behavior = "LOG"
+  }
+  
+  catalog_target {
+    database_name = "${aws_glue_catalog_database.test_1.name}"
+    tables = ["${aws_glue_catalog_table.test_1.name}"]
+  }
+
+  catalog_target {
+    database_name = "${aws_glue_catalog_database.test_2.name}"
+    tables = ["${aws_glue_catalog_table.test_2.name}"]
+  }
+
+  configuration = <<EOF
+{
+  "Version":1.0,
+  "Grouping": {
+    "TableGroupingPolicy": "CombineCompatibleSchemas"
+  }
+}
+EOF
+}
+`, rName, rName, rName, rName)
```

Comment:
Nit: Go's fmt verbs support indexing, e.g. `%[1]q` so repeated arguments are not necessary. 👍